modules/examples/personas/personas.py

```diff
@@ -0,0 +1,610 @@
+                    effect=iam.Effect.ALLOW,
+                    actions=[
+                        "sagemaker:DescribeModel",
+                        "sagemaker:DescribeEndpoint",
+                        "sagemaker:DescribeEndpointConfig",
+                        "sagemaker:DescribeTrainingJob",
+                        "sagemaker:DescribeHyperParameterTuningJob",
+                        "sagemaker:DescribeProcessingJob",
+                        "sagemaker:DescribeTransformJob",
+                        "sagemaker:ListModels",
+                        "sagemaker:ListEndpoints",
+                        "sagemaker:ListEndpointConfigs",
+                        "sagemaker:ListTrainingJobs",
+                        "sagemaker:ListHyperParameterTuningJobs",
+                        "sagemaker:ListProcessingJobs",
+                        "sagemaker:ListTransformJobs",
+                    ],
+                    resources=[f"arn:{Aws.PARTITION}:sagemaker:{Aws.REGION}:{Aws.ACCOUNT_ID}:model/*"],
+                ),
+                iam.PolicyStatement(
+                    effect=iam.Effect.ALLOW,
+                    actions=[
+                        "s3:GetObject",
+                        "s3:ListBucket",
+                    ],
+                    resources=[
+                        f"arn:{Aws.PARTITION}:s3:::{s3_bucket_prefix}*/*",
+                        f"arn:{Aws.PARTITION}:s3:::{s3_bucket_prefix}*",
+                    ],
+                ),
+                iam.PolicyStatement(
+                    effect=iam.Effect.ALLOW,
+                    actions=[
+                        "glue:GetTable",
+                        "glue:GetDatabase",
+                        "glue:GetJob",
+                        "glue:GetTrigger",
+                        "glue:GetCrawler",
+                        "glue:GetDevEndpoint",
+                    ],
+                    resources=[
+                        f"arn:{Aws.PARTITION}:glue:{Aws.REGION}:{Aws.ACCOUNT_ID}:crawler/*",
+                        f"arn:{Aws.PARTITION}:glue:{Aws.REGION}:{Aws.ACCOUNT_ID}:job/*"
+                        f"arn:{Aws.PARTITION}:glue:{Aws.REGION}:{Aws.ACCOUNT_ID}:catalog/*",
+                        f"arn:{Aws.PARTITION}:glue:{Aws.REGION}:{Aws.ACCOUNT_ID}:database/*",
+                        f"arn:{Aws.PARTITION}:glue:{Aws.REGION}:{Aws.ACCOUNT_ID}:table/*",
+                        f"arn:{Aws.PARTITION}:glue:{Aws.REGION}:{Aws.ACCOUNT_ID}:devEndpoint/*",
+                    ],
+                ),
+                iam.PolicyStatement(
+                    effect=iam.Effect.ALLOW,
+                    actions=[
+                        "ecr:GetAuthorizationToken",
+                        "ecr:BatchCheckLayerAvailability",
+                        "ecr:GetDownloadUrlForLayer",
+                        "ecr:BatchGetImage",
+                    ],
+                    resources=[f"arn:{Aws.PARTITION}:ecr:{Aws.REGION}:{Aws.ACCOUNT_ID}:repository/*"],
```

Comment:
Minor nitpick: I noticed an inconsistency three different options being used to retreive the region/account all throughout this module:
1. `AWS.REGION`
2. `env.region`
3. `this.region`

I think `this.region` (along with `this.account` and `this.partition`) would be better.